x/mint/keeper/keeper.go

```diff
@@ -121,6 +115,30 @@ func (k Keeper) SetParams(ctx sdk.Context, params types.Params) {
 	k.paramSpace.SetParamSet(ctx, &params)
 }
 
+// GetInflationTruncationDelta returns the truncation delta.
+// Panics if key is invalid.
+func (k Keeper) GetTruncationDelta(ctx sdk.Context, moduleAccountName string) (sdk.Dec, error) {
+	storeKey, err := getTruncationStoreKeyFromModuleAccount(moduleAccountName)
+	if err != nil {
+		return sdk.Dec{}, err
+	}
+	return osmoutils.MustGetDec(ctx.KVStore(k.storeKey), storeKey), nil
+}
+
+// SetTruncationDelta sets the truncation delta, returning an error if any. nil otherwise.
```

Comment:
```suggestion
// SetTruncationDelta sets the truncation delta, returning an error if any. nil otherwise.
// Only developer vesting and mint module accounts are allowed.
// Returns error if invalid module account.
```